tests/test_adapted_attributes.py

```diff
@@ -0,0 +1,153 @@
+import os
+import pytest
+import tempfile
+import datajoint as dj
+from datajoint.errors import ADAPTED_TYPE_SWITCH, FILEPATH_FEATURE_SWITCH
+import networkx as nx
+from itertools import zip_longest
+from . import schema_adapted
+from .schema_adapted import Connectivity, Layout
+from . import PREFIX, S3_CONN_INFO
+
+
+@pytest.fixture
+def adapted_graph_instance():
+    yield schema_adapted.GraphAdapter()
```

Comment:
Can't we just use `schema_adapted.GraphAdapter()` directly instead of accessing it through a fixture?

Reply:
We probably could, yes.